### PR TITLE
Create Drip (Actions) Destination documentation

### PR DESCRIPTION
**Pull request by jacobpmeyer**
### Proposed changes

Adding docs for the Drip Actions Destination

### Merge timing
Whenever works for y'all

### CODE REVIEW

File: src/connections/destinations/catalog/actions-drip/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+--
+title: Drip (Actions) Destination
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-drip/index.md:3]:
```suggestion
--
title: Drip (Actions) Destination
id: 673b62169b3342fbe0fc28da
---
```

File: src/connections/destinations/catalog/actions-drip/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+{% include content/plan-grid.md name="actions" %}
+
+[Drip](https://www.getdrip.com){:target="_blank”} is a nurture marketing platform Empowering B2C SMBs to convert long-sales cycle prospects into lifelong buyers with sophisticated and personalized marketing automation.
+
+This destination is maintained by Drip. For any issues with the destination, [contact their Support team](mailto:support@drip.com).
+
+> (delete after reading) The section below explains how to enable and configure the destination. Include any configuration steps not captured below. For example, obtaining an API key from your platform and any configuration steps required to connect to the destination.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-drip/index.md:11]:
```suggestion
```

File: src/connections/destinations/catalog/actions-drip/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Getting started
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Drip".
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-drip/index.md:15]:
```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Drip (Actions)".
```

File: src/connections/destinations/catalog/actions-drip/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+2. Select Drip and click **Add Destination**.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-drip/index.md:16]:
```suggestion
2. Select Drip (Actions) and click **Add Destination**.
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
just a couple changes, otherwise looks good

**Comment by jacobpmeyer** [comment]
Thanks @tcgilbert! I pushed up those changes just now

**Review by pwseg — APPROVED** [review]

**Pull request merged**